test/gurka/test_shortcut.cc

```diff
@@ -74,6 +74,42 @@ TEST(Shortcuts, CreateInvalid) {
   EXPECT_ANY_THROW(gurka::findEdgeByNodes(graph_reader, layout, "C", "A"));
 }
 
+// Here no shortcuts are created.
+// The grid size should be greater than the max length that allows shortcuts to be created
+TEST(Shortcuts, CreateTooLong) {
+  constexpr double gridsize = 2000000;
+
+  const std::string ascii_map = R"(
+      A--B--C--D----E
+  )";
+
+  const gurka::ways ways = {
+      {"AB",
+       {{"highway", "primary"},
+        {"name", "Independence Avenue"},
+        {"maxspeed:forward", "80"},
+        {"maxspeed:backward", "80"}}},
+      {"BC",
+       {{"highway", "primary"},
+        {"name", "Independence Avenue"},
+        {"maxspeed:forward", "80"},
+        {"maxspeed:backward", "80"}}},
+  };
+
+  const auto layout = gurka::detail::map_to_coordinates(ascii_map, gridsize);
+  auto map = gurka::buildtiles(layout, ways, {}, {}, "test/data/gurka_openlrjoiner_shortcut_speed");
+
+  baldr::GraphReader graph_reader(map.config.get_child("mjolnir"));
+
+  // check that there are no shortcut edges
+  auto shortcut_edge = std::get<1>(gurka::findEdgeByNodes(graph_reader, layout, "A", "C"));
```

Comment:
does that mean you tuned the test map in a way that you'll get a shortcut from A to C right? that'd be wonderful. but shouldn't there then also be a shortcut from C -> E? if not, is it possible to tune the map even further, so that the whole thing is too long, but we do get 2 shortcuts out of it in total?

ah wait.. you didn't specify the other ways in the map.. you only did `AB` and `BC`, the others don't even exist as ways as far as the map is concerned.